Draft cp release note for drupal contrib module (#9526)
pantheon-systems/documentation: source/releasenotes/2025-04-18-cp-drupal-contrib.md

---
title: "Content Publisher Drupal Module v1.0 now available"
published_date: "2025-04-18"
categories: [content-publisher, drupal, modules]
---

> Today we are releasing v1.0 of [Pantheon Content Publisher's Drupal Module](https://www.drupal.org/project/pantheon_content_publisher) - a new contrib module for Drupal sites. This new module replaces [the previous module](https://github.com/pantheon-systems/pcc-drupal-module) that was used by earlier participants of [the private beta program](https://pantheon.io/platform/content-publisher?_gl=1*pi6niv*_gcl_au*Mjc4NDY3NTIzLjE3NDQ4OTk4NDY.*_ga*ODM4NjY0NDYwLjE3NDQ4OTk4NDc.*_ga_CPJLBDH983*MTc0NDkwMzA3OS4yLjEuMTc0NDkwMzEwNi4wLjAuMA..).

Reviewer: ```suggestion
Today we are releasing v1.0 of [Pantheon Content Publisher's Drupal module](https://www.drupal.org/project/pantheon_content_publisher) - a new contrib module for Drupal sites. This new module replaces [the previous module](https://github.com/pantheon-systems/pcc-drupal-module) that was used by earlier participants of [the private beta program](https://pantheon.io/platform/content-publisher?_gl=1*pi6niv*_gcl_au*Mjc4NDY3NTIzLjE3NDQ4OTk4NDY.*_ga*ODM4NjY0NDYwLjE3NDQ4OTk4NDc.*_ga_CPJLBDH983*MTc0NDkwMzA3OS4yLjEuMTc0NDkwMzEwNi4wLjAuMA..).
```

## What's new?

This release marks a complete redesign of the Drupal integration for Content Publisher, and offers new functionality: 

> * Manages content as standard Drupal entities, rather than relying on Drupal views for rendering.

Reviewer: ```suggestion
* Manages content as standard Drupal entities, rather than relying on Drupal Views for rendering.
```

* Metadata fields are now converted into standard Drupal configured fields.

> * Drupal Search support has been added, allowing Content Publisher content to be searched by Drupal with blended results alongside other Drupal content.

Reviewer: ```suggestion
* Search API support has been added, allowing Content Publisher content to be searched by Drupal with blended results alongside other Drupal content.
```

> * Smart Components are now fieldable Drupal entities and no longer rely on Drupal Single Directory Components (SDC).

Reviewer: I think implementors still could use SDC to then them the Drupal entities.

```suggestion
* Smart Components from Google Docs get represented as fieldable Drupal entities.
```

* Increased reliability for real-time previews.
* Improved configuration panel.

## Learn More
While this module is now available on Drupal.org, it still requires access to the private beta program in order to be leveraged.

* [Request beta access](https://pantheon.io/platform/content-publisher?_gl=1*pi6niv*_gcl_au*Mjc4NDY3NTIzLjE3NDQ4OTk4NDY.*_ga*ODM4NjY0NDYwLjE3NDQ4OTk4NDc.*_ga_CPJLBDH983*MTc0NDkwMzA3OS4yLjEuMTc0NDkwMzEwNi4wLjAuMA..)
* Docs: [Content Publisher for Drupal: Getting Started Guide](https://docs.content.pantheon.io/pantheon-content-publisher-for-drupal)

> * Blog: Enhanced Drupal Integration for Content Publisher

Reviewer: ```suggestion
* Blog: [Enhanced Drupal Integration for Content Publisher](https://pantheon.io/blog/enhanced-drupal-integration-content-publisher)
```
